routes/routes.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const jsonParser = require('body-parser').json();
+const Wizard = require(`${__dirname}/../models/wizards.js`);
+const Film = require('../models/films.js');
+const sendMessage = require('../lib/sendMessage.js');
+
+const Router = module.exports = express.Router();
+
+Router.post('/wizards', jsonParser ,(req, res, next) => {
+    const newWizard = new Wizard(req.body);
+    newWizard.save().then(message => res.send(message))
+    .catch(err => next({error: err}));
+});
+
+Router.get('/wizards/:id', (req, res, next) => {
+    Wizard.findOne({_id: req.params.id})
+    .then(wizard => res.send(wizard))
+    .catch(err => next({error: err}));
+});
+
+Router.get('/wizards', (req, res) => {
+    Wizard.find(req.query || {})
+        .then(wizards => res.send('We got all the wizards correctly'))
+        .catch(err => res.send('Nope'));
+});
+
+Router.put('/wizards/:id',jsonParser,  (req, res, next) => {
+    if (typeof req.body['_id'] !== 'undefined') delete req.body._id;
+    Wizard.findOneAndUpdate({_id: req.params.id}, req.body)
+      .then(data => res.send(data))
+      .catch(err => next({error: err}));
+});
+
+Router.patch('/wizards/:id', jsonParser ,(req, res, next) => {
+    if (typeof req.body._id !== 'undefined') delete req.body._id;
+    Wizard.findOneAndUpdate({_id: req.params.id}, {$set: req.body}).then(data => res.send('Patched!')).catch(err => next({error: err}));
+});
+
+Router.delete('/wizards/:id', (req, res, next) => {
+    Wizard.remove({_id: req.params.id}).then(res.send('Wizard Deleted!')).catch(err => next({error: err}));
+});
+
+Router.get('/films', (req, res) => {
```

Comment:
films and wizards should be on separate routers, basically you should have a single router per resource.